app/controllers/api/v1/trucking/driver_commissions_history_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    module Trucking
+      # Driver Commission History Api endpoint
+      class DriverCommissionsHistoryController < ApplicationController
+        # GET /driver_commission_history.json
+        def index
+          respond_to do |format|
+            format.json { render :json => DriverCommissionHistoryAdapter.all }
+          end
+        end
+
+        # GET /driver_commission_history#by_paid_date
```

Comment:
Should read the same as the route, as you have in the index action above... so for instance it might be:

`GET /driver_commission_history/by_paid_date/2015-01-01.json`